specs/paymentrequest.html

```diff
@@ -284,16 +284,77 @@
           payment app.
         </div>
         <div class="note">
-          <p>The <code>supportedMethods</code> sequence contains the <a>payment method identifiers</a>
-          for the <a>payment methods</a> that the merchant web site accepts.</p>
+
+          <p>The <code>details</code> contains the core payment request data that will be passed on to the user selected
+            <a>payment app</a>.</p>
+
+          <p>The <code>amount</code> sequence contains the amount (possibly in different currencies) that the website
+            is requesting to be paid.</p>
+
+          <p>The <code>supportedMethods</code> sequence contains the <a>payment methods</a> that the merchant web
+            site accepts. Each <a>payment method</a> is linked to one or more <a>payment method identifiers</a>.</p>
+
+          <p>The <code>identifers</code> sequence contains the <a>payment method identifiers</a> that identify the
+          <a>payment methods</a> the web site supports and for which the associated <code>data</code> is part of the
```

Comment:
> We can make a separate PR to take "data" out if that's what the group wants to do later on.

Rather than hold up this PR I'd request that we do this.